Data/models/DataController.swift

```diff
@@ -41,18 +40,6 @@ public class DataController: NSObject {
         return mom
     }()
     
-    // Internal scope so unit tests can recreate the DB stack as needed.
-    override init() {
```

Comment:
This initalizer was called before migration was fully completed, and on first launch it was using the old documents directory. After removing it all should be working fine.

This was done to fix intermittent bug with data tests, but so far these tests are not failing, I will keep an eye on it